notes/10-auth-service.md

```diff
@@ -2,14 +2,18 @@
 
 Services allow state and functionality (i.e., regular functions, actions) to be shared across various parts of an Ember app.
 
-In our case, there are various things that may need to "see" or "use" authentication concerns (i.e., a `currentUser`).
+<!-- Explain what services are -->
+
+In our case, there are various things that may need to "see" or "use" authentication concerns (i.e., a `currentUser`):
+
+<!-- Clearify "authentication concerns" - state it differently  -->
 
 - Creating a new chat message
 - Preventing unauthenticated users from entering the app
 - Logging in
 - Logging out
 
-We could use a component for this, but it would increase the complexity of our templates, and would involve passing extra args through the component tree.
+We could use a component for this, but it would increase the complexity of our templates, and would involve passing extra arguements through the component tree.
```

Comment:
```suggestion
We could use a component for this, but it would increase the complexity of our templates and would involve passing extra named args (the things that look like `{{ @firstName }}`) through the component tree.
```